Action_inputs.md

```diff
@@ -0,0 +1,135 @@
+## Action inputs
+
+### Inputs specific to set cluster context
+<table>
+  <thead>
+    <tr>
+      <th>Action inputs</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>method</code><br/>Method</td>
+    <td>(Optional) Acceptable values: kubeconfig/service-account. Default value: kubeconfig</td>
+  </tr>
+  <tr>
+    <td><code>kubeconfig</code><br/>Kubectl config</td>
+    <td>(Relevant for kubeconfig approach) Contents of the configuration file to be used with kubectl (e.g. can be pulled from a secret)</td>
+  </tr>
+  <tr>
+    <td><code>context</code><br/>Context</td>
+    <td>(Relevant for kubeconfig approach) Context to be used within the provided kubeconfig file</td>
+  </tr>
+  <tr>
+    <td><code>k8s-url</code><br/>API server URL</td>
```

Comment:
you can have the 'API server URL' in a sperate column and name it appropriately

Reply:
I replaced it with the 'API server address' as this term is used on Azure Kubernetes Services portal.